app/components/free-response-form/component.js

```diff
@@ -0,0 +1,67 @@
+import Ember from 'ember';
+
+const formObject = Ember.Object.create({
+    "schema": {
```

Comment:
Sorry JS style is not enabled on this repo, but please:
- remove quotes from key names of JS Objects
- use single quotes for strings